.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+
+before_script:
+  - npm install -g aglio
+
+script:
+  - cd docs
+  - aglio -i ./apiary.apib -o apis.html
+
+deploy:
+  local_dir: ${TRAVIS_BUILD_DIR}/docs
+  target_branch: gh-pages
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+#  on:
```

Comment:
While having this commented out in forks may help testing the script, I think it'd be safer to limit this to changes to master. What do you think?

Reply:
Yes! I was just testing for my own repo - I'll uncomment it.